src/main/java/com/fundynamic/d2tm/game/map/renderer/MapCellViewportRenderer.java

```diff
@@ -26,17 +27,17 @@ public MapCellViewportRenderer(Map map, int tileHeight, int tileWidth, Vector2D
         cellsThatFitVertically = (windowDimensions.getY() / tileHeight) + 1;
     }
 
-    public void render(Image imageToDrawOn, Vector2D viewingVector, Renderer<MapCell> renderer) throws SlickException {
-        int startCellX = (viewingVector.getX() / tileWidth);
-        int startCellY = (viewingVector.getY() / tileHeight);
+    public void render(Image imageToDrawOn, Vector2f viewingVector, Renderer<MapCell> renderer) throws SlickException {
+        int startCellX = (int)(viewingVector.getX() / tileWidth);
```

Comment:
Vector2D wraps `Vector2f` so you could actually change this back to `Vector2D` and use the `getX` and `getY` methods? 

Although then you also would need some way to get the 'original' values (with float). So its a PITA either way.